src/component/registry/bpmn-elements-registry.ts

```diff
@@ -178,14 +165,23 @@ export class BpmnElementsRegistry {
    * @param classNames The name of the class(es) to remove from the BPMN element(s)
    */
   toggleCssClasses(bpmnElementIds: string | string[], classNames: string | string[]): void {
-    // TODO duplication with addCssClasses (only the call to removeClassNames or addClassNames or toggleClasses differ)
+    this.updateCssClasses(bpmnElementIds, classNames, this.cssRegistry.toggleClasses.bind(this.cssRegistry));
+  }
+
+  private updateCssClasses(
+    bpmnElementIds: string | string[],
+    classNames: string | string[],
+    cssClassesUpdaterFunction: (bpmnElementId: string, classNames: string[]) => boolean,
```

Comment:
Last try: `cssClassesUpdaterFunction` --> `cssClassesUpdater`? We usually don't put type in variable name

Reply:
I agree we do not do it,
Here, however, it can be a useful indicator that it is a callable param.
Maybe we can replace Function by Callback so param name becomes - cssClassesUpdaterCallback?

Or we simply rely on that: it is clear enough that we have a type: `() => boolean` that indicates it should be called?

Reply:
This is a private method, so no need to spend to much time on naming debate. 😸 
So I let you decide what is the best according to you.

Reply:
yes, true story